src/helpers/eventInstances/daily.ts

```diff
@@ -0,0 +1,50 @@
+import type mongoose from "mongoose";
+import type {
+  InterfaceEvent,
+  InterfaceOrganization,
+  InterfaceUser,
+} from "../../models";
+import { Event } from "../../models";
+import type { MutationCreateEventArgs } from "../../types/generatedGraphQLTypes";
+import { eachDayOfInterval, format } from "date-fns";
+
+interface InterfaceRecurringEvent extends MutationCreateEventArgs {
+  startDate: Date;
+  creatorId: mongoose.Types.ObjectId;
+  admins: mongoose.Types.ObjectId[];
+  organization: mongoose.Types.ObjectId;
+}
+
+export async function generateEvents(
+  args: Partial<MutationCreateEventArgs>,
+  currentUser: InterfaceUser,
+  organization: InterfaceOrganization,
+  session: mongoose.ClientSession
+): Promise<InterfaceEvent[]> {
+  const recurringEvents: InterfaceRecurringEvent[] = [];
+  const { data } = args;
+
+  const startDate = new Date(data?.startDate);
+  const endDate = new Date(data?.endDate);
```

Comment:
What is the `endDate` is undefined? How many events would then be created? Also if I add a huge end date, let's say `100 years`, then I would be adding `36500` events into the database at one go, that too for one event for just one user. This will completely blow up the database if the organizations start to use this feature leniently. How would you account for these issues?